Does not install on linux

> for dir in `echo m68k-palmos arm-palmos multi-bfd doc include tools | sed 's/make//'`; do \
>   (cd ${dir} && make DESTDIR="" install) || exit 1; \
> done
> make[1]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos'
> for dir in `echo binutils gcc295 gdb crt libc libm | sed 's/make//'`; do \
>   (cd ${dir} && make DESTDIR="" install) || exit 1; \
> done
> make[2]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils'
> /bin/sh ../../../prc-tools-2.3/binutils/mkinstalldirs /usr/local /usr/local
> make[3]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/intl'
> if test "@PACKAGE@" = "gettext" \
>    && test '' = 'intl-compat.o'; then \
>   if test -r ../../../../prc-tools-2.3/binutils/intl/../../mkinstalldirs; then \
>     ../../../../prc-tools-2.3/binutils/intl/../../mkinstalldirs /usr/local/lib /usr/local/include; \
>   else \
>     ../../../../prc-tools-2.3/binutils/intl/mkinstalldirs /usr/local/lib /usr/local/include; \
>   fi; \
>   /bin/sh /home/aladds/prc-tools-remix/prc-tools-2.3/binutils/install-sh -c -m 644 intlh.inst /usr/local/include/libintl.h; \
>   /bin/sh /home/aladds/prc-tools-remix/prc-tools-2.3/binutils/install-sh -c -m 644 libintl.a /usr/local/lib/libintl.a; \
> else \
>   : ; \
> fi
> if test "@PACKAGE@" = "gettext"; then \
>   if test -r ../../../../prc-tools-2.3/binutils/intl/../../mkinstalldirs; then \
>     ../../../../prc-tools-2.3/binutils/intl/../../mkinstalldirs /usr/local/share/gettext/intl; \
>   else \
>     ../../../../prc-tools-2.3/binutils/intl/mkinstalldirs /usr/local/share/gettext/intl; \
>   fi; \
>   /bin/sh /home/aladds/prc-tools-remix/prc-tools-2.3/binutils/install-sh -c -m 644 VERSION /usr/local/share/gettext/intl/VERSION; \
>   dists="ChangeLog Makefile.in linux-msg.sed po2tbl.sed.in xopen-msg.sed gettext.h gettextP.h hash-string.h libgettext.h loadinfo.h bindtextdom.c dcgettext.c dgettext.c gettext.c finddomain.c loadmsgcat.c localealias.c textdomain.c l10nflist.c explodename.c intl-compat.c cat-compat.c"; \
>   for file in $dists; do \
>     /bin/sh /home/aladds/prc-tools-remix/prc-tools-2.3/binutils/install-sh -c -m 644 ../../../../prc-tools-2.3/binutils/intl/$file /usr/local/share/gettext/intl/$file; \
>   done; \
> else \
>   : ; \
> fi
> make[3]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/intl'
> make[3]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/libiberty'
> make[4]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/libiberty/testsuite'
> make[4]: Nothing to be done for 'all'.
> make[4]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/libiberty/testsuite'
> /bin/sh ../../../../prc-tools-2.3/binutils/libiberty/../mkinstalldirs /usr/local/lib
> /bin/sh /home/aladds/prc-tools-remix/prc-tools-2.3/binutils/install-sh -c -m 644 libiberty.a /usr/local/lib/libiberty.an
> ( cd /usr/local/lib ; ranlib libiberty.an )
> mv -f /usr/local/lib/libiberty.an /usr/local/lib/libiberty.a
> if test -n ""; then \
>   case "" in \
>     /*)    thd=;; \
>     *)     thd=/usr/local/include/;; \
>   esac; \
>   /bin/sh ../../../../prc-tools-2.3/binutils/libiberty/../mkinstalldirs ${thd}; \
>   for h in ../../../../prc-tools-2.3/binutils/libiberty/../include/ansidecl.h ../../../../prc-tools-2.3/binutils/libiberty/../include/demangle.h ../../../../prc-tools-2.3/binutils/libiberty/../include/dyn-string.h ../../../../prc-tools-2.3/binutils/libiberty/../include/fibheap.h ../../../../prc-tools-2.3/binutils/libiberty/../include/floatformat.h ../../../../prc-tools-2.3/binutils/libiberty/../include/hashtab.h ../../../../prc-tools-2.3/binutils/libiberty/../include/libiberty.h ../../../../prc-tools-2.3/binutils/libiberty/../include/objalloc.h ../../../../prc-tools-2.3/binutils/libiberty/../include/partition.h ../../../../prc-tools-2.3/binutils/libiberty/../include/safe-ctype.h ../../../../prc-tools-2.3/binutils/libiberty/../include/sort.h ../../../../prc-tools-2.3/binutils/libiberty/../include/splay-tree.h ../../../../prc-tools-2.3/binutils/libiberty/../include/ternary.h; do \
>     /bin/sh /home/aladds/prc-tools-remix/prc-tools-2.3/binutils/install-sh -c -m 644 $h ${thd}; \
>   done; \
> fi
> make[4]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/libiberty/testsuite'
> make[4]: Nothing to be done for 'install'.
> make[4]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/libiberty/testsuite'
> make[3]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/libiberty'
> make[3]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes'
> Making install in po
> make[4]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes/po'
> make[4]: Nothing to be done for 'install'.
> make[4]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes/po'
> make[4]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes'
> make[5]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes'
> make[5]: Nothing to be done for 'install-exec-am'.
> make[5]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes'
> make[4]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes'
> make[3]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/opcodes'
> make[3]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/bfd'
> Making install in doc
> make[4]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/bfd/doc'
> make[5]: Entering directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/bfd/doc'
> make[5]: Nothing to be done for 'install-exec-am'.
> make[5]: Nothing to be done for 'install-data-am'.
> make[5]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/bfd/doc'
> makeinfo --no-split -I ../../../../../prc-tools-2.3/binutils/bfd/doc ../../../../../prc-tools-2.3/binutils/bfd/doc/bfd.texinfo
> ./elf.texi:1: warning: @section missing argument
> ./bfdt.texi:325: raising the section level of @subsubsection which is too low
> ./init.texi:5: raising the section level of @subsubsection which is too low
> ./archive.texi:54: raising the section level of @subsubsection which is too low
> ./format.texi:27: raising the section level of @subsubsection which is too low
> ./reloc.texi:497: raising the section level of @subsubsection which is too low
> ./core.texi:8: raising the section level of @subsubsection which is too low
> ./opncls.texi:5: raising the section level of @subsubsection which is too low
> ./libbfd.texi:10: raising the section level of @subsubsection which is too low
> ./cache.texi:14: raising the section level of @subsubsection which is too low
> ./elf.texi:13: raising the section level of @subsubsection which is too low
> Makefile:290: recipe for target 'bfd.info' failed
> make[4]: *** [bfd.info] Error 1
> make[4]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/bfd/doc'
> Makefile:948: recipe for target 'install-recursive' failed
> make[3]: *** [install-recursive] Error 1
> make[3]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils/bfd'
> Makefile:1662: recipe for target 'install-bfd' failed
> make[2]: *** [install-bfd] Error 2
> make[2]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos/binutils'
> Makefile:120: recipe for target 'install' failed
> make[1]: *** [install] Error 1
> make[1]: Leaving directory '/home/aladds/prc-tools-remix/build/m68k-palmos'
> Makefile:124: recipe for target 'install' failed
> make: *** [install] Error 1
> aladds@anoia:~/prc-tools-r

--- a/README.adoc
+++ b/README.adoc
@@ -1,5 +1,7 @@
-prc-tools remix
+prc-tools-remix
 ~~~~~~~~~~~~~~~
+
+image:https://travis-ci.org/jichu4n/prc-tools-remix.svg?branch=master["Build Status", link="https://travis-ci.org/jichu4n/prc-tools-remix"]
 
 Overview
 ^^^^^^^^
@@ -14,10 +16,10 @@ toolchain for https://en.wikipedia.org/wiki/Motorola_68000_series[m68k] and
 https://en.wikipedia.org/wiki/ARM_architecture[ARM], as well as Palm OS-specific
 utilities.
 
-As the Palm OS platform faded into oblivion over the past decade, prc-tools
-appears to be have been unmaintained since 2003 and no longer compiles under
-recent versions of GCC and Clang. This repo includes a number of patches to
-enable it to build and run on modern Linux and macOS systems.
+As the Palm OS platform faded into oblivion, prc-tools appears to be have been
+unmaintained since 2003 and no longer compiles under recent versions of GCC and
+Clang. This repo includes a number of patches to enable it to build and run on
+modern Linux and macOS systems.
 
 Building
 ^^^^^^^^
@@ -30,12 +32,12 @@ To build from source on Linux, e.g., on Ubuntu:
 [source,bash]
 ----
 sudo apt-get install \
-    texinfo \
-    flex \
-    bison \
-    gperf
+    build-essential autoconf automake \
+    libncurses-dev \
+    flex bison \
+    texinfo gperf
 
-git clone git@github.com:jichu4n/prc-tools-remix.git
+git clone https://github.com/jichu4n/prc-tools-remix.git
 cd prc-tools-remix
 
 mkdir build && cd build
@@ -43,11 +45,13 @@ mkdir build && cd build
     --enable-targets=m68k-palmos,arm-palmos \
     --enable-languages=c,c++ \
     --disable-nls \
-    --with-palmdev-prefix=/opt/palmdev \
-    --host=i686-linux-gnu
+    --build=i686-linux-gnu \
+    --host=i686-linux-gnu \
+    --prefix=/usr \
+    --with-palmdev-prefix=/opt/palmdev
 # Use "env SHELL=/bin/sh make" if using a non-POSIX shell like fish
 make
-sudo make install
+sudo make MAKEINFO=true install
 ----
 
 macOS
@@ -69,7 +73,7 @@ from http://brew.sh[Homebrew] / Xcode:
 
 [source,bash]
 ----
-git clone git@github.com:jichu4n/prc-tools-remix.git
+git clone https://github.com/jichu4n/prc-tools-remix.git
 cd prc-tools-remix
 
 mkdir build && cd build
@@ -77,11 +81,12 @@ mkdir build && cd build
     --enable-targets=m68k-palmos,arm-palmos \
     --enable-languages=c,c++ \
     --disable-nls \
-    --with-palmdev-prefix=/usr/local/opt/palmdev \
-    --host=i686-apple-darwin
+    --build=i686-apple-darwin \
+    --host=i686-apple-darwin \
+    --with-palmdev-prefix=/usr/local/opt/palmdev
 # Use "env SHELL=/bin/sh make" if using a non-POSIX shell like fish
 make
-make install
+make MAKEINFO=true install
 ----
 
 Usage

